TESTS/configs/baremetal.json

```diff
@@ -56,6 +56,9 @@
         },
         "DISCO_L475VG_IOT01A": {
             "target.c_lib": "small"
+        },
+        "LPC1114": {
+            "target.c_lib": "small"
```

Comment:
Since this board is Mbed 2 board, we should set `"target.c_lib": "small"` in `targets.json` instead of here.

Reply:
Thanks. I removed the `c_lib` here and moved to `targets.json`.